batch-cluster/cloudformation.yml

```diff
@@ -19,17 +19,18 @@ Parameters:
     Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
     Default: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
 
-Outputs:
+  ProductBucket:
```

Comment:
`Bucket` or `ContentBucket` might be better, I think it'll eventually hold browse images and maybe other files